### PR TITLE
pacific: mgr/devicehaelth: extract and present SSD wear

### PR DESCRIPTION
**Pull request by tchaikov**
backport of #39365

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: qa/tasks/mgr/dashboard/helper.py [hunk 1]
```diff
@@ -646,7 +646,7 @@ def __new__(cls, sub_elems, allow_unknown=False, none=False, unknown_schema=None
         'dev': str,
         'path': str
     }))
-}))
+}, allow_unknown=True))
```
Comment (nizamial09) [inline qa/tasks/mgr/dashboard/helper.py:649]: Possible Conflict with this PR https://github.com/ceph/ceph/pull/39596/files

Comment (tchaikov) [reply qa/tasks/mgr/dashboard/helper.py:649]: @nizamial09 how can a change backported from master conflicts with its original change?

**Review by nizamial09 — APPROVED** [review]

**Review by alfonsomthd — APPROVED** [review]

**Comment by yuriw** [comment]
@tchaikov pls rebase and merge at will it was approved by @neha-ojha

**Comment by tchaikov** [comment]
@yuriw ack. 

rebased.

**Review by liewegas — APPROVED** [review]

**Comment by smithfarm** [comment]
@tchaikov The PR description says this is a backport of https://github.com/ceph/ceph/pull/39365 but that PR has 5 commits, while this one has 11. Where do the other 6 commits come from? Would you recommend cherry-picking those to octopus in https://github.com/ceph/ceph/pull/40761 ?

**Pull request merged**